README.md

```diff
@@ -14,7 +14,7 @@ Specification for binding services to runtime applications running in Kubernetes
 
 *  Need a consistent way to bind k8s application to services (applications, databases, event streams, etc)
 *  A standard / spec / RFC will enable adoption from different service providers
-*  Cloud Foundry has done this well.  The equivalent is not available for k8s.
+*  Cloud Foundry has done this well using injection of a [VCAP_SERVICES environment variable](https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES). The equivalent is not available for k8s.
```

Comment:
I was actually referring to this spec: https://github.com/buildpacks/rfcs/blob/master/text/0012-service-binding.md  Which advocates against env vars: https://github.com/buildpacks/rfcs/blob/master/text/0012-service-binding.md#how-it-works

Reply:
I wasn't too sure what you were referencing actually, hence the need for a reference. 😄 
Do you want me to edit the PR with the proper link? Otherwise, feel free to close it.

Reply:
cool :)  Ya, that would be great if you could edit the PR with the correct link.

Reply:
Updated.